utilities/config/config.go

```diff
@@ -142,6 +142,14 @@ func SetAPIURL(value string) {
 	constants.ConfigOverride["URL"] = value
 }
 
+func SetDNSAddresses(value string) {
```

Comment:
@LLParse I'd like to contain as much of the windows-specific logic to _windows.go files as possible. I think adding this to the config file just sort-of pollutes the linux code base and makes it confusing because the config property is not used by linux (correct me if I'm wrong on that).

So, can you rip this out of this file and remove the property you added to main.go and simply look for this as an environment variable in the _windows.go files where you need it?

I could see how removing it as a flag in main.go could hurt usability. So, we can talk about that part if you like.